src/main/java/in/projecteka/consentmanager/user/UserService.java

```diff
@@ -96,22 +96,17 @@
         return signupService.getMobileNumber(otpVerification.getSessionId())
                 .switchIfEmpty(Mono.error(ClientError.networkServiceCallFailed()))
                 .flatMap(mobileNumber -> {
-            OtpAttempt attempt = OtpAttempt.builder()
-                    .sessionId(otpVerification.getSessionId())
-                    .identifierType("MOBILE")
-                    .identifierValue(mobileNumber)
-                    .action(OtpAttempt.Action.OTP_SUBMIT_REGISTRATION)
-                    .build();
-            return otpAttemptService.validateOTPSubmission(attempt)
-                    .then(otpServiceClient.verify(otpVerification.getSessionId(), otpVerification.getValue(), () -> otpAttemptService.createOtpAttemptFor(
-                            otpVerification.getSessionId(),
-                            "",
-                            "MOBILE", mobileNumber,
-                            OtpAttempt.AttemptStatus.FAILURE,
-                            OtpAttempt.Action.OTP_SUBMIT_REGISTRATION)))
-                    .then(otpAttemptService.removeMatchingAttempts(attempt))
-                    .then(signupService.generateToken(otpVerification.getSessionId()));
-        });
+                    OtpAttempt.OtpAttemptBuilder builder = OtpAttempt.builder()
+                            .sessionId(otpVerification.getSessionId())
+                            .identifierType("MOBILE")
+                            .identifierValue(mobileNumber)
```

Comment:
remove the inline hardcoded identifierType.

Reply:
I don't think we can do much about it as the identifier type is not coming as part of request body. One thing that we can do is create a Enum or a Constant.

Reply:
done.